Description: Mention the possibility to add /version=1.1 to the ServerName
 configuration in various places:
 - man client.conf
 - doc/help/ref-client-conf.html
Bug-Debian: http://bugs.debian.org/704238
Bug-Debian: http://bugs.debian.org/711192
Author: Didier Raboud <odyx@debian.org>
Last-Update: 2013-06-06

--- a/man/client.conf.man.in
+++ b/man/client.conf.man.in
@@ -40,12 +40,12 @@
 ("name@server.example.com") for you. The default name is
 "@CUPS_DEFAULT_GSSSERVICENAME@".
 .TP 5
-ServerName hostname-or-ip-address[:port]
+ServerName hostname-or-ip-address[:port][/version=1.1]
 .TP 5
 ServerName /domain/socket
 .br
 Specifies the address and optionally the port to use when connecting to the
-server. \fBNote: Not supported on OS X 10.7 or later.\fR
+server. The IPP version (2.0 by default, can be 1.1 or 1.0) can be specified to access older servers. \fBNote: Not supported on OS X 10.7 or later.\fR
 .TP 5
 User name
 .br
--- a/doc/help/ref-client-conf.html
+++ b/doc/help/ref-client-conf.html
@@ -56,6 +56,7 @@
 ServerName foo.bar.com
 ServerName 11.22.33.44
 ServerName foo.bar.com:8631
+ServerName foo.bar.com/version=1.1
 </PRE>
 
 <H3>Description</H3>
@@ -64,6 +65,8 @@
 
 <P>The default port number is 631 but can be overridden by adding a colon followed by the desired port number to the value.</P>
 
+<P>The default IPP version is 2.0 but can be overriden by adding a slash followed by <CODE>/version=</CODE> and the desired IPP version (can be 1.0 or 1.1).</P>
+
 <P>The default is to use the local server ("<VAR>localhost</VAR>") or domain socket, if so configured.</P>
 
 <BLOCKQUOTE><B>Note:</B>
